### ISSUE TITLE
npm install error on node v12

### ISSUE THREAD

**Issue by DavidBurela** [pokusew/node-pcsclite#24]
I can install correctly on node v10, but v12 throws the below error

```
npm install @pokusew/pcsclite

> @pokusew/pcsclite@0.5.2 install /mnt/c/source/github/nfc-pcsc/node_modules/@pokusew/pcsclite
> node-gyp rebuild

make: Entering directory '/mnt/c/source/github/nfc-pcsc/node_modules/@pokusew/pcsclite/build'
  CXX(target) Release/obj.target/pcsclite/src/addon.o
In file included from ../src/addon.cpp:1:0:
../src/pcsclite.h:31:30: error: ‘v8::Handle’ has not been declared
         static void init(v8::Handle<v8::Object> target);
                              ^~~~~~
../src/pcsclite.h:31:36: error: expected ‘,’ or ‘...’ before ‘<’ token
         static void init(v8::Handle<v8::Object> target);
                                    ^
In file included from ../src/addon.cpp:2:0:
../src/cardreader.h:90:30: error: ‘v8::Handle’ has not been declared
         static void init(v8::Handle<v8::Object> target);
                              ^~~~~~
../src/cardreader.h:90:36: error: expected ‘,’ or ‘...’ before ‘<’ token
         static void init(v8::Handle<v8::Object> target);
                                    ^
../src/cardreader.h:123:20: error: ‘Handle’ in namespace ‘v8’ does not name a template type
         static v8::Handle<v8::Value> CreateBufferInstance(char* data, unsigned long size);
                    ^~~~~~
../src/addon.cpp:4:19: error: variable or field ‘init_all’ declared void
 void init_all(v8::Handle<v8::Object> target) {
                   ^~~~~~
../src/addon.cpp:4:19: error: ‘Handle’ is not a member of ‘v8’
../src/addon.cpp:4:36: error: expected primary-expression before ‘>’ token
 void init_all(v8::Handle<v8::Object> target) {
                                    ^
../src/addon.cpp:4:38: error: ‘target’ was not declared in this scope
 void init_all(v8::Handle<v8::Object> target) {
                                      ^~~~~~
../src/addon.cpp:4:38: note: suggested alternative: ‘tzset’
 void init_all(v8::Handle<v8::Object> target) {
                                      ^~~~~~
                                      tzset
In file included from ../../../nan/nan.h:54:0,
                 from ../src/pcsclite.h:4,
                 from ../src/addon.cpp:1:
../src/addon.cpp:9:23: error: ‘init_all’ was not declared in this scope
 NODE_MODULE(pcsclite, init_all)
                       ^
/home/david/.node-gyp/12.2.0/include/node/node.h:556:36: note: in definition of macro ‘NODE_MODULE_X’
       (node::addon_register_func) (regfunc),                          \
                                    ^~~~~~~
../src/addon.cpp:9:1: note: in expansion of macro ‘NODE_MODULE’
 NODE_MODULE(pcsclite, init_all)
 ^~~~~~~~~~~
../src/addon.cpp:9:23: note: suggested alternative: ‘initstate’
 NODE_MODULE(pcsclite, init_all)
                       ^
/home/david/.node-gyp/12.2.0/include/node/node.h:556:36: note: in definition of macro ‘NODE_MODULE_X’
       (node::addon_register_func) (regfunc),                          \
                                    ^~~~~~~
../src/addon.cpp:9:1: note: in expansion of macro ‘NODE_MODULE’
 NODE_MODULE(pcsclite, init_all)
 ^~~~~~~~~~~
pcsclite.target.mk:125: recipe for target 'Release/obj.target/pcsclite/src/addon.o' failed
make: *** [Release/obj.target/pcsclite/src/addon.o] Error 1
make: Leaving directory '/mnt/c/source/github/nfc-pcsc/node_modules/@pokusew/pcsclite/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/david/.nvm/versions/node/v12.2.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:196:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:257:12)
gyp ERR! System Linux 4.4.0-18362-Microsoft
gyp ERR! command "/home/david/.nvm/versions/node/v12.2.0/bin/node" "/home/david/.nvm/versions/node/v12.2.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /mnt/c/source/github/nfc-pcsc/node_modules/@pokusew/pcsclite
gyp ERR! node -v v12.2.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @pokusew/pcsclite@0.5.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @pokusew/pcsclite@0.5.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by jimmythesaint82** [comment]
https://github.com/pokusew/node-pcsclite/pull/23

**Comment by pokusew** [comment]
@DavidBurela fixes released in version [0.6.0](https://github.com/pokusew/node-pcsclite/releases/tag/v0.6.0)

**Comment by DavidBurela** [comment]
I appreciate you taking the time to update the compatibility
